test/hotspot/jtreg/serviceability/jvmti/events/MonitorWaited/monitorwaited01/libmonitorwaited01.cpp

```diff
@@ -88,27 +93,13 @@ static void check_stack_trace(JNIEnv* jni, jthread thr) {
   err = jvmti->GetStackTrace(thr, 0, MAX_COUNT, frameInfo, &count);
   check_jvmti_status(jni, err, "event handler: error in JVMTI GetStackTrace call");
 
-  const int expected_count = 8;
-  const char* expected_methods[expected_count] = {"waitImpl", "wait", "wait", "run", "runWith", "run", "run", "enter"};
+  const int expected_count = 7;
 
   if (count != expected_count) {
-    LOG("Expected 8 methods in the stack but found %d", count);
+    LOG("Expected 7 methods in the stack but found %d", count);
     jni->FatalError("Unexpected method count");
   }
 
-  for (int idx = 0; idx < count; idx++) {
-    jclass declaringClass = nullptr;
-    char *clasSignature = nullptr;
-    char *methodName = nullptr;
-
-    err = jvmti->GetMethodName(frameInfo[idx].method, &methodName, nullptr, nullptr);
-    check_jvmti_status(jni, err, "event handler: error in JVMTI GetMethodName call");
-
-    if (strcmp(methodName, expected_methods[idx]) != 0) {
-      LOG("Expected method %s but found %s", expected_methods[idx], methodName);
-      jni->FatalError("Unexpected method found");
-    }
-  }
```

Comment:
Why can we not just update the names of the expected methods instead of not checking any of them?

Reply:
In the past, method names were updated. Ref: https://github.com/ibmruntimes/openj9-openjdk-jdk/pull/938.

Reply:
Is the list given in https://github.com/eclipse-openj9/openj9/pull/21590#issuecomment-2806374028 not what should always be expected?

Reply:
updated change to retain the method name check and match OpenJ9's stacktrace